requirements.txt

```diff
@@ -32,12 +32,12 @@ cffi==1.14.0
     # via cryptography
 chardet==3.0.4
     # via requests
-clickhouse-driver==0.1.4
+clickhouse-driver==0.2.0
     # via
     #   -r requirements.in
     #   aioch
     #   clickhouse-pool
-clickhouse-pool==0.4.0
+clickhouse-pool==0.4.3
```

Comment:
Can you remove this dep? We don't use it any more

Reply:
Looks like it didn't get removed after all?

Reply:
It's actually used - discussed in slack.